### PR TITLE
[POAE7-1456] Add Velox cbrt function to Omnisci

### PR DESCRIPTION
**Pull request by xieqi**

### CODE REVIEW

File: QueryEngine/CMakeLists.txt [hunk 1]
```diff
@@ -308,7 +308,18 @@ set(QUERY_ENGINE_LIBS
   ${Arrow_LIBRARIES}
 )
 
-list(APPEND QUERY_ENGINE_LIBS ${llvm_libs} ${ZLIB_LIBRARIES})
+find_library(GLOG glog)
+find_package(gflags COMPONENTS shared)
+find_library(FMT fmt)
+find_library(EVENT event)
+find_package(folly REQUIRED)
+set(FOLLY_WITH_DEPENDENCIES
```
Comment (winningsix) [inline QueryEngine/CMakeLists.txt:316]: @lousijiaintel FYI, this PR impacts dependencies for cicricle.

**Comment by winningsix** [comment]
Overall LGTM. @xieqi offline discussed, let's see what Omnisci plan looks like using functionOp. @yma11 can you help review on this also?

**Comment by yma11** [comment]
@winningsix for function translation, we will handle 3 cases:
1) Omnisci supported functions, including SQLOps and SQLAggOps, will translate them in exprs such as BinOper, UOper, AggExpr, etc.
2) extended functions which leverage existing FunctionExpr, will translate into FunctionExpr, like Cbrt(double) here.
3) for complicated functions, as addressed in [PR](https://github.com/Intel-bigdata/omniscidb/pull/48), which introduces a new expr type "VeloxFunctionExpr", will translate to VeloxFunctionExpr when deal with such functions.
@xieqi, please correct me if I misunderstand.

**Comment by yma11** [comment]
LGTM.

**Pull request merged**